### ISSUE TITLE
Please make the purchase message customisable

### ISSUE THREAD

**Issue by Formyndare** [thirdy/durian#37]
@<PLAYER> Hi, WTB your <"ITEM"> listed for <# CURRENCY> in <LEAGUE.>

Is what I'm reffering to - Unless I missed something and there is in fact a way to alter this?

**Comment by kyri0** [comment]
Hello, this will be made end-user avaible via javascprit eventually. For now, if you want to alter the message, you will have to download the the source code (avaiable in the release page), and edit the file where the WTB message is. After that, you would need to compile Durian on your own. If you know how to do this, you can find the WTB message file in:

https://github.com/thirdy/durian/blob/master/durian/src/main/java/qic/SearchPageScraper.java

Search for:

```
public String wtb() {
//            String mods = buildWTBModsMessage();
//            return String.format(
//                    "@%s Hi, I would like to buy your %s listed for %s in %s. With stats%s",
//                    getIgn(), getName(), getBuyout(), getLeague(), mods);
            return String.format(
                    "@%s Hi, WTB your \"%s\" listed for %s in %s league.",
                    ign, name, buyout, league);
        }
```

@thirdy will eventually make a better guide for this (if the update with the end-user edit option doesn't come first).

**Comment by Formyndare** [comment]
This is great info and a quick response, thank you!

**Comment by thirdy** [comment]
Implemented with 0.6, closing
